### PR TITLE
Updated Flink cli binaries.

### PR DESCRIPTION
**Pull request by joerg84**

### CODE REVIEW

**Comment by joerg84** [comment]
> Diffing package flink revision 4 revision 5
> 
> 
> Diffing File config.json:
> 
> 
> Diffing File marathon.json.mustache:
> 
> 
> Diffing File package.json:
> 
> 
> Diffing File resource.json:
> 21c21
> <           "url":"https://github.com/mesosphere/dcos-flink-service/blob/master/container/appmaster/cli/dcos-flink/artifacts/dcos-flink-service-darwin?raw=true"
> ---
> >           "url":"https://github.com/mesosphere/dcos-flink-service/raw/1.1/container/appmaster/cli/dcos-flink/artifacts/dcos-flink-service-darwin"
> 28c28
> <           "url":"https://github.com/mesosphere/dcos-flink-service/blob/master/container/appmaster/cli/dcos-flink/artifacts/dcos-flink-service-linux?raw=true"
> ---
> >           "url":"https://github.com/mesosphere/dcos-flink-service/raw/1.1/container/appmaster/cli/dcos-flink/artifacts/dcos-flink-service-linux"
> 35c35
> <           "url":"https://github.com/mesosphere/dcos-flink-service/blob/master/container/appmaster/cli/dcos-flink/artifacts/dcos-flink-service.exe?raw=true"
> ---
> >           "url":"https://github.com/mesosphere/dcos-flink-service/raw/1.1/container/appmaster/cli/dcos-flink/artifacts/dcos-flink-service.exe"

**Comment by joerg84** [comment]
The binaries themselves are the same and hence the hashes as well.

File: repo/packages/F/flink/5/resource.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+          "contentHash":[ { "algo":"sha256", "value":"b55ca85862e9123a0c3731441479eabc544d9900f5a17cdd0d1f24a7f4f3eac5" } ],
+          "kind":"executable",
+          "url":"https://github.com/mesosphere/dcos-flink-service/raw/1.1/container/appmaster/cli/dcos-flink/artifacts/dcos-flink-service-darwin"
+        }
+      },
+      "linux":{
+        "x86-64":{
+          "contentHash":[ { "algo":"sha256", "value":"9b13f0f22d4a6cbc48cf2d001ed509bc1f54015b385e5062112d28d673841c5a" } ],
+          "kind":"executable",
+          "url":"https://github.com/mesosphere/dcos-flink-service/raw/1.1/container/appmaster/cli/dcos-flink/artifacts/dcos-flink-service-linux"
```
Comment (OhRobin) [inline repo/packages/F/flink/5/resource.json]: 1.1

File: repo/packages/F/flink/5/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
+{
+    "packagingVersion": "3.0",
+
+    "name": "flink",
+    "version": "1.3.1-1.2",
```
Comment (OhRobin) [inline repo/packages/F/flink/5/package.json]: will keep the same package version?

File: repo/packages/F/flink/5/resource.json (hunk 1 above)
Comment (joerg84) [inline repo/packages/F/flink/5/resource.json]: 1.1 is correct

File: repo/packages/F/flink/5/package.json (hunk 2 above)
Comment (joerg84) [inline repo/packages/F/flink/5/package.json]: addressed, ty!

File: repo/packages/F/flink/5/package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
+{
+    "packagingVersion": "3.0",
+
+    "name": "flink",
+    "version": "1.3.1-1.2.1",
```
Comment (OhRobin) [inline repo/packages/F/flink/5/package.json]: that works for me

**Pull request merged**